var/spack/repos/builtin/packages/libuv/package.py

```diff
@@ -5,29 +5,40 @@
 class Libuv(AutotoolsPackage):
     """Multi-platform library with a focus on asynchronous IO"""
     homepage = "https://libuv.org"
-    url = "https://github.com/libuv/libuv/archive/v1.9.0.tar.gz"
+    url = "https://dist.libuv.org/dist/v1.44.1/libuv-v1.44.1-dist.tar.gz"
```

Comment:
Can you add a brief comment stating that the `*-dist` comes with a configure script already generated?

Reply:
done